tests/sources/xml/test_ead.py

```diff
@@ -1,10 +1,61 @@
+# ruff: noqa: E501
```

Comment:
Perhaps just a personal opinion but I think it's preferable to not ignore E501 for a module (or even a line really). For the `xml_string` var in `create_ead_source_record_stub`, you can use spaces instead of tabs and you can break up the long citation strings like this:

```
assert Ead.get_citation(source_record) == (
        "Charles J. Connick Stained Glass Foundation Collection, "
        "VC-0002, box X. Massachusetts Institute of Technology, "
        "Department of Distinctive Collections, Cambridge, Massachusetts."
    )
```

Reply:
Good idea! On a somewhat related note (and is also why I hesitated to reformat the strings):

When I save on an open XML file, I have a tool in my IDE that reformats the XML. It results in newlines being added throughout the XML file, which then causes the tests to fail. 

@ehanson8 @ghukill Do you think it's worth creating an issue to whitespace/newline-proof the XML transforms? 🤔

Reply:
We'd probably want to preserve whitespace inside an element (its text), as this is often intentional by the record creator.  I think it would fundamentally change the nature of the content if we introduce newline characters inside a long string of text for an element.

But reformatting the XML in a way that just moves elements around, but not element text, that seems okay / good.

Reply:
I think I'll forego creating an issue for now as the newlines are resulting from me not turning off the XML formatter in my IDE 😅 . 

Circling back to original comment, I removed the syntax to ignore `E501` errors and made updates accordingly!